src/dbally/views/structured.py

```diff
@@ -19,21 +17,18 @@
 
 class BaseStructuredView(BaseView):
     """
-    Base class for all structured [Views](../../concepts/views.md). All classes implementing this interface has\
+    Base class for all structured views. All classes implementing this interface has\
```

Comment:
nit: why remove the link?

Reply:
this link didn't work

Reply:
Just tested the link again (using the `mkdocs serve` command and looking at the description of the class under http://127.0.0.1:8000/reference/views/structured/) and the link works for me

Reply:
In the documents it works fine, but in the IDE it doesn't work. I wonder if we really need this if we are linking to the same resource referenced in the tips section.

Reply:
We have similar links to specific concepts all over the documentations - we specifically worked on adding them when originally writing the documentation, so that the users can quickly reference what a given concept is about. If you think we should remove them - we can discuss this. But I don't think quietly removing just a single instance makes sense.